src/library/driftory.ts

```diff
@@ -228,24 +231,28 @@ export default class Driftory {
   }
 
   goToFrame(index: number) {
-    var frame = this.frames[index];
-    var bufferFactor = 0.2;
-    var box = frame.clone();
-
-    box.width *= 1 + bufferFactor;
-    box.height *= 1 + bufferFactor;
-    box.x -= frame.width * bufferFactor * 0.5;
-    box.y -= frame.height * bufferFactor * 0.5;
-
-    this.viewer?.viewport.fitBounds(box);
+    if (this.getFrameIndex() !== index) {
+      var frame = this.frames[index];
+      var bufferFactor = 0.2;
+      if (frame) {
+        var box = frame.clone();
+
+        box.width *= 1 + bufferFactor;
+        box.height *= 1 + bufferFactor;
+        box.x -= frame.width * bufferFactor * 0.5;
+        box.y -= frame.height * bufferFactor * 0.5;
+
+        this.viewer?.viewport.fitBounds(box);
+      }
+    }
   }
 
   getFrameIndex() {
     return this.frameIndex;
   }
 
   figureFrameIndex() {
-    let bestIndex = -1;
+    let bestIndex = 0;
```

Comment:
I figure `0` (first frame of the comic) makes for a better default index than `-1` (???) or is there a specific reason you chose `-1` as the default index?

Reply:
Yes, -1 is meaningful. This function is used to tell you what you're currently focused on, and if you're not focused on any part of the comic, that's what -1 means. This guides the click interaction. I'll go ahead and change it back.

Was this just something you noticed, or was there a behavior you were trying to fix?